### PR TITLE
[16.0][IMP] contract and contract_variable_quantity: improve visual of contract lines

### PR DESCRIPTION
**Pull request by aliciagaarzo**
Working with the contracts you have found the following view for the contract lines:

Several changes have been made to improve the visual for the l contract lines, which is displayed as follows:

### CODE REVIEW

**Review by amarcosg — APPROVED** [review]
LGTM code review

File: contract_variable_quantity/views/abstract_contract_line.xml [hunk 1]
```diff
@@ -8,10 +8,8 @@
             ref="contract.contract_abstract_contract_line_form_view"
         />
         <field name="arch" type="xml">
-            <xpath expr="//label[@for='quantity']" position="before">
-                <field name="qty_type" />
-            </xpath>
             <xpath expr="//field[@name='quantity']" position="after">
+                <field name="qty_type" />
```
Comment (pedrobaeza) [inline contract_variable_quantity/views/abstract_contract_line.xml:12]: Please don't change this position, as it doesn't make sense to put the quantity before the quantity type

Comment (aliciagaarzo) [reply contract_variable_quantity/views/abstract_contract_line.xml:12]: Absolutely right @pedrobaeza , I changed the position field to show it before quantity.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**